src/libraries/System.Numerics.Tensors/src/CompatibilitySuppressions.xml

```diff
@@ -127,4 +127,130 @@
     <Right>lib/netstandard2.0/System.Numerics.Tensors.dll</Right>
     <IsBaselineSuppression>true</IsBaselineSuppression>
   </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0017</DiagnosticId>
+    <Target>M:System.Numerics.Tensors.Tensor.GreaterThanOrEqualAll``1(System.Numerics.Tensors.ReadOnlyTensorSpan{``0}@,``0)$0</Target>
```

Comment:
this is because of https://github.com/dotnet/runtime/pull/107097#pullrequestreview-2267229024
@ericstj I'd have expected APICompat to fail on that PR already, any idea why it only shows up here?

Reply:
It's brand new API so we are allowed to break it.  In 9.0 we're comparing to the 8.0.0 packages.  Here in 10.0 we're comparing to the 9.0 preview7 packages.